UploadAndAnnotateImages/README.md

```diff
@@ -8,6 +8,7 @@ We recommend taking the [Quick Start: Lightning Web Components](https://trailhea
 
 - Set up your Salesforce DX environment
 - Set up Visual Studio Code
+- Make sure your org is updated to version 244 and higher
```

Comment:
@karinlackrif The release name that will be communicated with customers externally will be `Summer '23` not `244` so you might want to rename.